collisions group id

On a fresh install with centos7 (7.6.1810)  which uses the uses infosys and edginfo users, I find collisions in the groupid.

[INFO] running component: accounts
---------------------------------------------------------
[INFO] System has 33 accounts in 57 groups
[WARN] No groups found for root in the system. Assigning default ones: root, adm, bin, daemon, sys, disk
[INFO] Changing gid of group nagios to 76389
[ERROR] Collision found between groups ssh_keys and infosys for id 997
[ERROR] Collision found between groups edginfo and input for id 999
[ERROR] System would be inconsistent. Leaving without changing anything
[INFO] configure on component accounts executed, 3 errors, 1 warnings
[INFO] EVENT: accounts modified file /etc/login.defs

--- a/users/edginfo.pan
+++ b/users/edginfo.pan
@@ -1,21 +1,16 @@
-
 unique template users/edginfo;
 
-# -----------------------------------------------------------------------------
-# accounts
-# -----------------------------------------------------------------------------
-include { 'components/accounts/config' };
+include 'components/accounts/config';
 
-include { 'users/infosys' };
+include 'users/infosys';
 
-"/software/components/accounts/groups/edginfo" =
-  nlist("gid", 999);
+"/software/components/accounts/groups/edginfo" = dict("gid", 999);
 
-"/software/components/accounts/users/edginfo" = nlist(
-  "uid", 999,
-  "groups", list("edginfo","infosys"),
-  "comment","EDG Info",
-  "shell", "/bin/bash",
-  "homeDir", "/home/edginfo"
+"/software/components/accounts/users/edginfo" = dict(
+    "uid", 999,
+    "groups", list("edginfo", "infosys"),
+    "comment", "EDG Info",
+    "shell", "/bin/bash",
+    "homeDir", "/home/edginfo",
 );
 

--- a/users/edginfo.pan
+++ b/users/edginfo.pan
@@ -4,7 +4,7 @@ include 'components/accounts/config';
 
 include 'users/infosys';
 
-"/software/components/accounts/groups/edginfo" = dict("gid", 999);
+"/software/components/accounts/groups/edginfo" = dict("gid", 1999);
 
 "/software/components/accounts/users/edginfo" = dict(
     "uid", 999,

--- a/users/infosys.pan
+++ b/users/infosys.pan
@@ -1,12 +1,5 @@
-
 unique template users/infosys;
 
-# -----------------------------------------------------------------------------
-# infosys is a group shared by all accounts collecting information
-# -----------------------------------------------------------------------------
-include { 'components/accounts/config' };
+include 'components/accounts/config';
 
-"/software/components/accounts/groups/infosys" =
-  nlist("gid", 997);
-
-
+"/software/components/accounts/groups/infosys" = dict("gid", 997);

--- a/users/infosys.pan
+++ b/users/infosys.pan
@@ -2,4 +2,4 @@ unique template users/infosys;
 
 include 'components/accounts/config';
 
-"/software/components/accounts/groups/infosys" = dict("gid", 997);
+"/software/components/accounts/groups/infosys" = dict("gid", 1997);

